tensorflow/compiler/mlir/tosa/transforms/legalize_utils.cc

```diff
@@ -154,8 +160,14 @@ Value buildRescaleToInt32(PatternRewriter& rewriter, Operation* op,
   assert(input_type);
   auto output_type = input_type.clone(rewriter.getI32Type());
 
+#if TFLITE_SINGLE_ROUNDING
+  bool double_round = false;
```

Comment:
Could we localize this checking on TFLITE_SINGLE_ROUNDING to 1 spot? (the less #if #else on it the better for me/easier to not forget). E.g., could this be inside buildRescale and we don't need to pass in double_round anywhere?

Reply:
Hi Jacques,

Thanks for the review! I agree with what you want and I actually tried moving that IFELSE to buildRescale but it will break other legalizations (e.g., Reduce_MEAN) since there're some cases we do need double_round=false even thought the global SINGLE_ROUNDING is set to false.

Jerry